yellowfinLib.py

```diff
@@ -359,15 +366,25 @@ def makePOSfileFromRINEX(
         https://rtkexplorer.com/pdfs/manual_demo5.pdf
     """
     # arguments for command here: https://rtkexplorer.com/pdfs/manual_demo5.pdf
+
     sp3 = kwargs.get("sp3", "")
-    freq = kwargs.get("freq", 3)
 
     logging.debug(
         f"converting {os.path.basename(roverObservables)} using RTKLIB: Q=1:fix,2:float,3:sbas,4:dgps,5:single,6:ppp"
     )
-    os.system(
-        f"./{executablePath} -o {outfname} -t -u -f {freq} {roverObservables} {baseObservables} {navFile} {sp3}"
-    )
+
+    # If using a Windows machine
+    if os.name == 'nt':
```

Comment:
Maybe a better way to do this is make the string that's called By os.system in the if/then statement.